package.json

```diff
@@ -45,10 +45,14 @@
     "jest": "26.6.0"
   },
   "devDependencies": {
+    "cookie": "^0.4.1",
     "eslint": "^7.22.0",
+    "eslint-config-airbnb": "^18.2.1",
     "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-config-prettier": "^8.1.0",
     "eslint-plugin-import": "^2.22.1",
     "eslint-plugin-jest": "^24.1.0",
+    "eslint-plugin-prettier": "^3.3.1",
```

Comment:
what is the purpose of these new packages?

Reply:
pre-commit gave me this errors:
 ` ESLint couldn't find the config "airbnb" to extend from.`
 ` ESLint couldn't find the config "prettier" to extend from.`
 ` ESLint couldn't find plugin prettier to extend from.`
when I searched about it I found that 
 "eslintConfig": {
    "extends": [
      "airbnb",
     "prettier"
    ]}
when it searched for extends in node modules it doesn't found it so I installed it and the error disappeared